src/Microsoft.OData.Client/DataServiceContext.cs

```diff
@@ -696,8 +696,10 @@ public virtual EntityTracker EntityTracker
         public virtual bool EnableWritingODataAnnotationWithoutPrefix { get; set; }
 
         /// <summary>
-        /// Indicates whether a Where clause that just compares the key property generates a $filter query option.
+        /// When true, a Where clause that just compares the key property generates a $filter query option, otherwise a key segment is generated.

```

Comment:
```suggestion
        /// When true, a Where clause that has only the key property in the predicate generates a $filter query option, otherwise a key segment is generated.
```